src/bindings/python/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenVINO Python API
+
+OpenVINO Python API is a part of OpenVINO library. The component is responsible for:
+
+* Bindings of OpenVINO - allowing users to utilize OpenVINO library in their Python code. Python API provides bindings to basic and advanced APIs from OpenVINO Runtime.
+* Extending OpenVINO with pythonic features - on top of direct translations from C++, Python API component:
+    * Adds specific extensions to support numpy-based data.
+    * Provides support for external frameworks inputs.
+    * Provides shortcuts and helpers with more pythonic design.
+    * Allows to apply advanced concepts like shared memory to take full advantage of OpenVINO.
```

Comment:
Do we have any code styles and something like that for python?

Reply:
I will add that if necessary. However we are following adjusted style checks and everything is described in [contributing.md](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/docs/contributing.md), so link against it is enough?

Reply:
Linked with current state.